### ISSUE TITLE
[BUG] Soulbound rune does not work with ClayTech installed

### ISSUE THREAD

**Issue by VoltroXP** [ClayCoffee/ClayTech#111]
** Describe 描述 **
In Slimefun, without ClayTech installed the Soubound rune works fine, but with ClayTech installed it does not.

** To Reproduce 如何复现 **
1. Get a soulbound rune
2. Make sure you have ClayTech installed
3. Drop the soulbound rune on an item
4. Nothing happens

** Running Environment 运行环境 **
Using latest versions of Claytech and Slimefun from https://thebusybiscuit.github.io/builds/

** Screenshots 截图 **
I don't have screenshots but there is a great video regarding this issue that was opened on Slimefun found here https://github.com/Slimefun/Slimefun4/issues/2788

**Comment by PiggiesGoSqueal** [comment]
Just worth noting no developer (I've seen) will accept 'Using the latest versions' as a helpful answer. You should include exact versions in bug reports. 'latest' changes all the time and even if they determine which was the latest when you made your post- there's the risk of human error. In other words, the version being 'latest' when you made your post might not be what your server actually uses.

TLDR: In future when reporting bugs to any developer do not say you're using the 'latest versions' say the actual versions. Also a latest.log is also helpful for devs to find info like server version, plugin versions, etc.
